karma.conf.js

```diff
@@ -3,6 +3,7 @@ const path = require('path');
 module.exports = function(config) {
     const coverageReporters = [];
     const coverageLoaders = [];
+    let tsConfigFile = 'tsconfig.json'
```

Comment:
Please update the commit message to one of the following propositions:
- [IMP] Karma: add sourcemap for coverage
- [IMP] Karma: add back sourcemap for coverage